doc/dnf5.conf.5.rst

```diff
@@ -1358,6 +1358,15 @@ Variables Directories
 ``Distribution Variables Directory``
     /usr/share/dnf5/vars.d/
 
+
+Source and debuginfo repository names
+=====================================
+
+For a given repository with an ID in the form "<ID>-rpms", its corresponding source repository is expected to have an ID in the form "<ID>-source-rpms" and debuginfo repository an ID in the form "<ID>-debug-rpms". Otherwise (if the repository doesn't have the suffix "-rpms"), the source repository is expected to have an ID in the form "<ID>-source" and debuginfo repository an ID in the form "<ID>-debuginfo".
```

Comment:
Please change parenthesized note `if the repository doesn't have the suffix "-rpms"` into `if the repository identifier doesn't have the "-rpms" suffix`. I.e. add "identifier and move "-rpms" ahead because of "the" article.